files/recipe.yaml

```diff
@@ -80,10 +80,10 @@
     pip:
       name: git+https://github.com/packit-service/packit.git
       executable: pip3
-  - name: Install ogr from git master
-    pip:
-      name: git+https://github.com/packit-service/ogr.git
-      executable: pip3
+#  - name: Install ogr from git master
+#    pip:
+#      name: git+https://github.com/packit-service/ogr.git
+#      executable: pip3
```

Comment:
Just because of failing imports?

Reply:
Yes, because of this: https://github.com/packit-service/packit/pull/425